<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StoreWindow</class>
 <widget class="QMainWindow" name="StoreWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Здравствуйте, </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="username">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>#USER</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="logout">
        <property name="text">
         <string>Выйти</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="goodsTable">
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="addGood">
        <property name="text">
         <string>Добавить товар</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="editGood">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Редактировать товар</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeGood">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Удалить товар</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="info">
        <property name="text">
         <string>Справка</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>addGood</sender>
   <signal>clicked()</signal>
   <receiver>StoreWindow</receiver>
   <slot>onAddGoodClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>584</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>600</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editGood</sender>
   <signal>clicked()</signal>
   <receiver>StoreWindow</receiver>
   <slot>onEditGoodClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>591</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>601</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeGood</sender>
   <signal>clicked()</signal>
   <receiver>StoreWindow</receiver>
   <slot>onRemoveGoodClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>443</x>
     <y>583</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>599</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onAddGoodClicked()</slot>
  <slot>onEditGoodClicked()</slot>
  <slot>onRemoveGoodClicked()</slot>
 </slots>
</ui>
